src/sentry/api/helpers/group_index/delete.py

```diff
@@ -79,8 +80,9 @@ def delete_group_list(
         },
     )
 
-    # Tell seer to delete grouping records for these groups
-    may_schedule_task_to_delete_hashes_from_seer(error_ids)
```

Comment:
The logic to fetch group hashes will be moved from `may_schedule_task_to_delete_hashes_from_seer` will be moved into `delete_group_hashes`.